### DISCUSSION TITLE
Welcome to formation-git-hub Discussions!

### DISCUSSION THREAD

**Discussion by Gaetanbrl** [jdev-org/formation-git-hub#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pierrejego** [comment]
Bonjour à tous et bonne formation !

**Reply by catmorales** [reply]
Merci ! Bonne journée à toi aussi !
